### DISCUSSION TITLE
Add beta tester to closed Android app testing?

### DISCUSSION THREAD

**Discussion by yazinsai** [fastlane/fastlane#19016] [Q&A] [answered]
In iOS, I can run `fastlane pilot add email@invite.com -g Beta` to add a user to the TestFlight testers list.

What's the equivalent for adding a user to an Android Internal Testing track?

**Comment by raldred** [comment, marked as answer]
It's not implemented. 
But could be as the publisher service has the following methods:

[get_edit_tester](https://googleapis.dev/ruby/google-api-client/latest/Google/Apis/AndroidpublisherV3/AndroidPublisherService.html#get_edit_tester-instance_method)
[update_edit_tester](https://googleapis.dev/ruby/google-api-client/latest/Google/Apis/AndroidpublisherV3/AndroidPublisherService.html#update_edit_tester-instance_method)
[patch_edit_tester](https://googleapis.dev/ruby/google-api-client/latest/Google/Apis/AndroidpublisherV3/AndroidPublisherService.html#patch_edit_tester-instance_method)

**Reply by yazinsai** [reply]
Thanks! Is there a way to call these methods via Fastlane, or must I access this using Ruby with the `google-api-ruby-client` gem?

**Reply by raldred** [reply]
No its not implemented within fastlane you'd have to implement it yourself. Should be relatively straightforward.
